Specs/Sequence/Japanese/URLModel.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "Input": "はmicrosoft.comです",
```

Comment:
Here I think it's OK to leave only one Japanese case for now. The goal of this spec is to show that Japanese will reuse the Chinese model and things will work.